src/fetch/payouts.py

```diff
@@ -436,13 +436,6 @@ def construct_payout_dataframe(
         config.reward_config.reward_token_address.address
     )
 
-    # 7. Missing service fee is treated as new solver
```

Comment:
This was removed since the meaning of missing entries changed, right? Before it was an indication of missing data for new solvers. Now it could also just be a solver who is not in the COW bonding pool.

Not important for today's payouts, but it might be useful to reactivate a check restricted to missing service fee status for solvers in the COW bonding pool.

Reply:
Yes, indeed, the previous query was supposed to return an entry for every single active (vouched) solver, while this one only cares about CoW DAO bonding pool solvers. So if we want to bring back the check, we need to restrict it to the CoW DAO solvers